nameko/click_cli/myclick/argsopts.py

```diff
@@ -0,0 +1,68 @@
+"""Custom click option and argument decorators
```

Comment:
`myclick` is not an convention. In fact, typical click based code does not move that stuff often out of the main module, but here I was thinking of possible reuse (I plan to do one myself for our project possibly cloning the CLI with modified logging configuration).

Searching for proper name `click` was a candidate, which sounded too similar to original click. If you have better idea, I will follow that. `customclick`?

If you think, options with arguments shall share the same module with click types, let me know and I will do it that way.

Reply:
How about the following:

* `nameko/[click_]cli/click_arguments.py`
* `nameko/[click_]cli/click_options.py`
* `nameko/[click_]cli/click_paramtypes.py`

(Adding `[click_]` here because I'm assuming we will rename this package to simply `cli` when the old one is replaced)

This seems very clear and explicit to me.

Reply:
I will do it according to your proposal.

As we are almost done with all comments to this PR, I would now remove the original argparse based code and do the renaming at once.

Reply:
Modules renamed.

additionally, I renamed `run.py` to `do_run.py` and `shell.py` to `do_shell.py`.

Remaining steps is now to remove the argparse implementation and rename click_cli to cli